README.md

```diff
@@ -2,3 +2,338 @@
 
 This is a DRAFT standard for building APIs in Hackney council - it is subject
 to review and change.
+
+Many of the standards set out in this document are based on the
+[White House Web API Standards](https://github.com/WhiteHouse/api-standards),
+with some specifics from the
+[Google JSON StyleGuide](https://google.github.io/styleguide/jsoncstyleguide.xml#Date_Property_Values)
+
+## Data Properties
+
+- Property names should be in lower camel case
+  - Good: `"contactTelephoneNumber"`
```

Comment:
Should these be left unquoted?

e.g. `contactTelephoneNumber` vs `"contactTelephoneNumber"`

Reply:
Yup - probably better: I've added a new commit